src/components/pages/History/ComputeJobs.tsx

```diff
@@ -69,50 +94,91 @@ const columns = [
+  const { accountId } = useWeb3()
   const [jobs, setJobs] = useState<ComputeJobMetaData[]>()
   const [isLoading, setIsLoading] = useState(false)
 
+  const { data } = useQuery(getComputeOrders, {
+    variables: {
+      user: accountId.toLowerCase()
+    }
+  })
+
   useEffect(() => {
-    async function getTitle(did: string) {
-      const ddo = await ocean.metadataCache.retrieveDDO(did)
-      const metadata = ddo.findServiceByType('metadata')
-      return metadata.attributes.main.name
+    if (data === undefined || !config?.metadataCacheUri) return
+    const source = axios.CancelToken.source()
+
+    const didList = []
+    for (let i = 0; i < data.tokenOrders.length; i++) {
+      const newDid = web3.utils
+        .toChecksumAddress(data.tokenOrders[i].datatokenId.address)
+        .replace('0x', 'did:op:')
+      didList.push(newDid)
+    }
+    const queryDidList = JSON.stringify(didList)
+      .replace(/,/g, ' ')
+      .replace(/"/g, '')
+      .replace(/(\[|\])/g, '')
+      .replace(/(did:op:)/g, '0x')
+
+    const queryDid = {
+      page: 1,
+      offset: 100,
+      query: {
+        query_string: {
+          query: queryDidList,
+          fields: ['dataToken']
+        }
+      }
+    }
+
+    async function getAssetDetails() {
+      const assetList = []
+      const result = await queryMetadata(
+        queryDid,
+        config.metadataCacheUri,
+        source.token
+      )
+      for (let i = 0; i < result.results.length; i++) {
+        assetList.push({
+          did: result.results[i].id,
+          assetName: result.results[i].service[0].attributes.main.name,
+          type: result.results[i].service[1].type
+        })
+      }
+      return assetList
     }
 
     async function getJobs() {
       if (!ocean || !account) return
       setIsLoading(true)
       try {
-        const orderHistory = await ocean.assets.getOrderHistory(
-          account,
-          'compute',
-          100
-        )
         const jobs: ComputeJobMetaData[] = []
-
-        for (let i = 0; i < orderHistory.length; i++) {
-          const assetName = await getTitle(orderHistory[i].did)
-          const computeJob = await ocean.compute.status(
-            account,
-            orderHistory[i].did,
-            undefined,
-            orderHistory[i].transactionHash,
-            false
-          )
-          computeJob.forEach((item) => {
-            jobs.push({
-              did: orderHistory[i].did,
-              jobId: item.jobId,
-              dateCreated: item.dateCreated,
-              dateFinished: item.dateFinished,
-              assetName: assetName,
-              status: item.status,
-              statusText: item.statusText,
-              algorithmLogUrl: '',
-              resultsUrls: []
+        const assets = await getAssetDetails()
+        for (let i = 0; i < assets.length; i++) {
+          if (assets[i].type === 'compute') {
+            const computeJob = await ocean.compute.status(
```

Comment:
somewhere within in this method the assets get lost, when doing `console.log(computeJob)` right after this, it's an empty array for me, where `assets` above actually has the assets:

<img width="503" alt="Screen Shot 2021-03-31 at 12 34 26" src="https://user-images.githubusercontent.com/90316/113131397-7172fc00-921d-11eb-9502-a4f5fe258713.png">

That `[zeroXTransformer] Input transformation failed.` implies we do something wrong in the parameters we pass to `ocean.compute.status`